apps/election-manager/src/data/defaultElection.json

```diff
@@ -17,6 +17,9 @@
       "state": "Estado de Muestra"
     }
   },
+  "ballotLayout": {
```

Comment:
should we add to the sample elections in /fixtures as well?

Reply:
Yes, definitely. I was getting tired and wanted to push this MVP. 😴